aries-site/src/examples/components/card/NavigationalAnatomy.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Diagram,
+  Grid,
+  Heading,
+  Paragraph,
+  Stack,
+} from 'grommet';
+import { Notes } from 'grommet-icons';
+import { Annotation } from '../../../layouts';
+
+const DashedBox = ({ ...rest }) => (
+  <Box
+    alignSelf="start"
+    border={{ style: 'dashed' }}
+    round="xxsmall"
+    {...rest}
+  />
+);
+
+const CardExample = ({ ...rest }) => (
+  <Card width="medium" onClick={() => {}} {...rest}>
+    <CardBody align="start" gap="medium">
+      <Box gap="xsmall">
+        <>
+          <Box pad={{ bottom: 'small' }}>
+            <DashedBox id="icon">
+              <Notes color="blue" />
+            </DashedBox>
+          </Box>
+          <DashedBox id="title">
+            <Heading level={3} margin="none" size="small">
+              Helpful Guides
+            </Heading>
+          </DashedBox>
+        </>
+        <DashedBox id="description">
+          <Paragraph margin="none">
+            Access step by step guides on getting the most out of your Greenlake
+            console.
+          </Paragraph>
+        </DashedBox>
+      </Box>
+      <DashedBox id="action" flex={false}>
```

Comment:
The buttons here don't seem to be getting a hover treatment.

Reply:
Fixed.